docs/src/main/sphinx/release/release-369.md

```diff
@@ -0,0 +1,163 @@
+# Release 369 (24 Jan 2022)
+
+## General
+
+* Add support for `Pacific/Kanton` time zone. ({issue}`10679`)
+* Display `Physical input read time` using most succinct time unit in `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`10576`)
+* Fine tune request retry mechanism in HTTP event listener. ({issue}`10566`)
+* Add support for using PostgreSQL and Oracle as backend database for resource
+  groups. ({issue}`9812`)
+* Remove unnecessary spilling configuration properties `spill-order-by` and
+  `spill-window-operator`. ({issue}`10591`)
+* Remove distinction between system and user memory to simplify
+  cluster configuration. The configuration property
+  `query.max-total-memory-per-node` is removed. Use `query.max-memory-per-node`
+  instead. ({issue}`10574`)
+* Use formatting specified in the SQL standard when casting `double` and `real`
+  values to `varchar` type. ({issue}`552`)
+* Add support for `ALTER MATERIALIZED VIEW ... SET PROPERTIES`. ({issue}`9613`)
+* Add experimental implementation of task level retries. This can be enabled by
+  setting the `retry-policy` configuration property or the `retry_policy`
+  session property to `task`. ({issue}`9818`)
+* Improve query wall time by splitting workload between nodes in a more balanced
+  way. Previous workload balancing policy can be restored via
+  `node-scheduler.splits-balancing-policy=node`. ({issue}`10660`)
+* Prevent hanging query execution on failures with `phased` execution policy.
+  ({issue}`10656`)
+* Catch overflow in decimal multiplication. ({issue}`10732`)
+* Fix `UnsupportedOperationException` in `max_by` and `min_by` aggregation.
+  ({issue}`10599`)
+* Fix incorrect results or failure when casting date to `varchar(n)` type.
+  ({issue}`552`)
+* Fix issue where the length of log file names grow indefinitely upon log
```

Comment:
Missing link to PR: https://github.com/trinodb/trino/pull/10738

Reply:
done